pypeit/calibrations.py

```diff
@@ -124,17 +143,28 @@ def __init__(self, fitstbl, par, spectrograph, caldir, qadir=None,
             os.makedirs(os.path.join(self.qa_path, 'PNGs'))
 
         # Attributes
-        self.calib_dict = {}
         self.det = None
         self.frame = None
         self.binning = None
 
+        self.shape = None
```

Comment:
Nice, maybe add some comments explaining what each attribute is, i.e. analogous to the data model in data container.

Reply:
It is in the doc string above.